packages/client/lib/miner/miner.ts

```diff
@@ -229,8 +296,12 @@ export class Miner {
     }
     if (interrupt) return
     // Build block, sealing it
-    const block = await blockBuilder.build()
-    this.config.logger.info(`Miner: Sealed block with ${block.transactions.length} txs`)
+    const block = await blockBuilder.build(this.nextSolution)
+    this.config.logger.info(
+      `Miner: Sealed block with ${block.transactions.length} txs${
```

Comment:
Same as above, cosmetic but aligns the logs with the consensus algorithm
```suggestion
his.config.logger.info(
      `Miner: ${this.config.chainCommon.consensusType === ConsensusType.ProofOfAuthority
        ? `Sealed`
        : `Mined`
      } block with ${block.transactions.length} txs${
        this.nextSolution ? ` (difficulty: ${block.header.difficulty})` : ''
      }`
    )
```